batchflow/models/eager_torch/layers/resize.py

```diff
@@ -0,0 +1,324 @@
+    squeeze_units : int or sequence of ints
+        Sizes of dense layers.
+    squeeze_activations : str or sequence of str
+        Activations of dense layers.
+    """
+    def __init__(self, inputs=None, ratio=4, squeeze_layout='Vfafa', squeeze_units=None, squeeze_activations=None):
+        from .conv_block import ConvBlock # can't be imported in the file beginning due to recursive imports
+        super().__init__()
+        in_units = get_shape(inputs)[1]
+        units = squeeze_units or [in_units // ratio, in_units]
+        activations = squeeze_activations or ['relu', 'sigmoid']
+
+        self.layer = ConvBlock(layout=squeeze_layout, units=units, activations=activations, inputs=inputs)
+
+
+    def forward(self, x):
+        x = self.layer(x)
+        return x.view(x.size(0), -1, 1, 1)
+
+
+
+class SideBlock(nn.Module):
+    """ Add side branch to a :class:`~.layers.ConvBlock`. """
+    def __init__(self, inputs=None, **kwargs):
+        from .conv_block import ConvBlock # can't be imported in the file beginning due to recursive imports
+        super().__init__()
+        self.layer = ConvBlock(inputs=inputs, **kwargs)
+
+    def forward(self, x):
+        return self.layer(x)
+
+
+
+class Combine(nn.Module):
+    """ Combine list of tensor into one.
+
+    Parameters
+    ----------
+    inputs : sequence of torch.Tensors
+        Tensors to combine.
+
+    op : str or callable
+        If callable, then operation to be applied to the list of inputs.
+        If 'concat', 'cat', '.', then inputs are concated along channels axis.
+        If 'sum', '+', then inputs are summed.
+        If 'mul', '*', then inputs are multiplied.
+        If 'avg', then inputs are averaged.
+        If 'softsum', '&', then every tensor is passed through 1x1 convolution in order to have
+        the same number of channels as the first tensor, and then summed.
+    """
+    @staticmethod
+    def concat(inputs):
+        return torch.cat(inputs, dim=1)
+
+    @staticmethod
+    def sum(inputs):
+        return torch.stack(inputs, dim=0).sum(dim=0)
+
+    @staticmethod
+    def mul(inputs):
+        """ Multiplication. """
+        result = 1
+        for item in inputs:
+            result = result * item
+        return result
+
+    @staticmethod
+    def mean(inputs):
+        return torch.mean(inputs)
+
+    @staticmethod
+    def softsum(self, inputs):
+        """ Softsum. """
+        #pylint: disable=bad-staticmethod-argument
+        inputs = [conv(tensor) for conv, tensor in zip(self.conv, inputs)]
+        return torch.stack(inputs, dim=0).sum(dim=0)
+
+    OPS = {
+        concat.__func__: ['concat', 'cat', '.'],
+        sum.__func__: ['sum', 'plus', '+'],
+        mul.__func__: ['multi', 'mul', '*'],
+        mean.__func__: ['average', 'avg', 'mean'],
+        softsum.__func__: ['softsum', '&'],
+    }
+
+    OPS = {alias: method for method, aliases in OPS.items() for alias in aliases}
+    ALL_OPS = list(OPS.keys())
```

Comment:
Do you need this list as a class variable? You use it only once, so you can calculate it there